symengine/cwrapper.cpp

```diff
@@ -1,4 +1,3 @@
-#include <cstdlib>
```

Comment:
Does that mean it's getting imported implicitly by another header or that it's not used at all?